python/cudf/cudf/core/dataframe.py

```diff
@@ -2392,7 +2392,6 @@ def join(
         method="hash",
     ):
         """Join columns with other DataFrame on index or on a key column.
-
```

Comment:
Why did we remove this linebreaks?

Reply:
Oops - I'll remove these accidental changes in `dataframe.py` :)